NimbleMedium/Sources/Presentation/Modules/Feeds/FeedsView.swift

```diff
@@ -7,64 +7,144 @@
 
 import SwiftUI
 import Refresh
+import ToastUI
 
 struct FeedsView: View {
 
     @ObservedViewModel private var viewModel: FeedsViewModelProtocol
+    
+    @State private var isFirstLoad: Bool = true
+    @State private var isRefeshing: Bool = false
+    @State private var hasMore: Bool = true
+    @State private var isLoadingMore: Bool = false
+    @State private var articles: [Article] = []
+    @State private var isErrorToastPresented = false
 
     var body: some View {
-        NavigationView {
-            ScrollView {
+        Content(view: self)
+            .binding()
+            .onAppear {
+                viewModel.input.refresh()
+            }
```

Comment:
can make this one line 😁
```suggestion
            .onAppear { viewModel.input.refresh() }
```